### ISSUE TITLE
[BUG] Secondary info not showing for Grid

### ISSUE THREAD

**Issue by e10kstarfire** [flixlix/power-flow-card-plus#279]
**Describe the bug**
Secondary info entity is not showing for the Grid item

**To Reproduce**
```
type: custom:power-flow-card-plus
watt_threshold: 999
clickable_entities: true
use_new_flow_rate_model: true
entities:
  fossil_fuel_percentage:
    display_zero: true
    state_type: percentage
    entity: sensor.co2_signal_grid_fossil_fuel_percentage
  home:
    entity: sensor.givtcp_ed2253gXXX_load_power
    override_state: true
  grid:
    display_state: one_way
    entity:
      consumption: sensor.givtcp_ed2253gXXX_import_power
      production: sensor.givtcp_ed2253gXXX_export_power
    secondary_info:
      entity: select.givtcp_ed2253gXXX_mode
  solar:
    entity: sensor.givtcp_ed2253gXXX_pv_power
  battery:
    display_state: one_way
    entity:
      consumption: sensor.givtcp_ed2253gXXX_discharge_power
      production: sensor.givtcp_ed2253gXXX_charge_power
    state_of_charge: sensor.givtcp_ed2253gXXX_soc
  individual1:
    entity: sensor.myenergi_eddi_power_charging
    icon: mdi:shower
    name: Hot Water
    display_zero: true
    secondary_info:
      entity: sensor.myenergi_eddi_status
max_expected_power: 5000
```

![image](https://github.com/flixlix/power-flow-card-plus/assets/23463652/8eab5ce9-2016-4ac8-a15a-5b772f6b0dd9)

**Comment by flixlix** [comment]
Fixed in #280 
Although I wouldn't call this a bug, since your entity probably has a state type of string. Up until this point only the individual entities supported strings instead of numbers in the `secondary_info`. You could also use templates before.

Will be available in the next version

**Comment by e10kstarfire** [comment]
Ah I see , thanks very much ☺️
